packages/odyssey-react-mui/src/Badge.tsx

```diff
@@ -80,17 +81,20 @@ const Badge = ({
       fontFamily: `${odysseyDesignTokens.TypographyFamilyMono}`,
       fontWeight: `${odysseyDesignTokens.TypographyWeightBodyBold}`,
       lineHeight: 1,
-    }),
-    [type, contentIsLongerThanOneChar, odysseyDesignTokens]
-  );
+      transitionDuration: `${odysseyDesignTokens.TransitionDurationMain}`,
+      transitionProperty: `background-color, color`,
+    };
 
-  const shouldHideBadge = badgeContent <= 0 || !badgeContent;
+    const shouldRenderBadge = badgeContent && badgeContent > 0;
 
-  if (shouldHideBadge) {
-    return null;
-  }
+    if (shouldRenderBadge) {
+      return <Box sx={badgeStyles}>{formattedContent}</Box>;
+    } else {
+      return null;
+    }
```

Comment:
We can simplify this to a single return with a ternary:
```suggestion
    return (
      hasNotificationCount
      ? <Box sx={badgeStyles}>{formattedContent}</Box>
      : null
    )
```